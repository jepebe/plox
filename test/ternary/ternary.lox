print 5==5? "then clause of ?" : "else clause of ?";
print 5!=5? "then clause of ?" : "else clause of ?";
print true ? false ? 'ignored' : 'selected' : 'ignored';
print false ? false ? 'ignored' : 'ignored' : true ? 'selected' : 'ignored';
print 5 ? '5 is truthy!' : '5 is falsey?';
print !5 ? '!5 is truthy?' : '!5 is falsey!';