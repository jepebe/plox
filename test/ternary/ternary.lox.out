then clause of ?
else clause of ?
selected
selected
5 is truthy!
!5 is falsey!
